fmriprep/utils/static_files.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
```

Comment:
This entire file reminds me to this other file: https://github.com/poldracklab/preprocessing-workflow/blob/master/fmriprep/data/utils.py